Libplanet/Blockchain/BlockChain.cs

```diff
@@ -90,6 +98,11 @@ internal BlockChain(IBlockPolicy<T> policy, IStore store, Guid id)
 
             _logger = Log.ForContext<BlockChain<T>>()
                 .ForContext("CanonicalChainId", Id);
+
+            if (Genesis is null)
```

Comment:
Is it even possible to be `null`?

Reply:
Yes, if you had never made the chain at the store, `Genesis` will be `null`.
It's same with `BlockChain<T>.Count == 0`.

Reply:
Is it still valid?

Reply:
What if `Genesis` is not `null` while `genesisBlock` is present at a time?